lucene/backward-codecs/src/test/org/apache/lucene/backward_codecs/lucene87/Lucene87RWCodec.java

```diff
@@ -37,6 +38,17 @@ public PostingsFormat getPostingsFormatForField(String field) {
           return defaultPF;
         }
       };
+  private final Mode mode;
+
+  public Lucene87RWCodec() {
+    super();
+    this.mode = Mode.BEST_COMPRESSION;
```

Comment:
I think we should default to `BEST_SPEED` instead of `BEST_COMPRESSION`, otherwise `BEST_SPEED` won't be exercised in unit tests. We could factor this like the other `Lucene86RWCodec` classes and just create a default `Lucene87StoredFieldsFormat` here?

Reply:
ok